services/webfinger/pkg/command/server.go

```diff
@@ -0,0 +1,142 @@
+package command
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/oklog/run"
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/configlog"
+	"github.com/owncloud/ocis/v2/ocis-pkg/version"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/config"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/config/parser"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/logging"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/metrics"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/relations"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/server/debug"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/server/http"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/service/v0"
+	"github.com/owncloud/ocis/v2/services/webfinger/pkg/tracing"
+	"github.com/urfave/cli/v2"
+)
+
+// Server is the entrypoint for the server command.
+func Server(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:     "server",
+		Usage:    fmt.Sprintf("start the %s service without runtime (unsupervised mode)", cfg.Service.Name),
```

Comment:
Interpreting this I have the feeling that this info should go into readme.md
This would also include that the readme misses a list of cli commands available

Reply:
that is a more common topic. every service can be started standalone.